### PR TITLE
Dash into develop

### PR DESCRIPTION
**Pull request by quinn-dougherty**
Moving forward, Dash PRs will be against `develop` branch.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,10 @@
--e .
```
Comment (jlubken) [inline requirements.txt]: I think we want to keep the setup.py from develop and add the dash dependencies to the setup.py

File: setup.py [hunk 2]
```diff
@@ -1,46 +1,45 @@
...
 __author__ = "Predictive Healthcare @ Penn Medicine"
 
 from os import path
-from setuptools import setup, find_packages, find_namespace_packages
 
+from setuptools import setup, find_namespace_packages
+
+CWD = path.abspath(path.dirname(__file__))
+
+with open(path.join(CWD, "README.md"), encoding="utf-8") as inp:
```
Comment (jlubken) [inline setup.py]: The setup.cfg file does this work for README.md

File: setup.py [hunk 3]
```diff
@@ -1,46 +1,45 @@
+# -*- coding: utf-8 -*-
 """Setup file for chime
 """
 __version__ = "1.1.2"
 __author__ = "Predictive Healthcare @ Penn Medicine"
 
 from os import path
-from setuptools import setup, find_packages, find_namespace_packages
```
Comment (jlubken) [inline setup.py]: develop version is cleaner here.

**Review by jlubken — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (quinn-dougherty) [reply requirements.txt]: done

File: setup.py (hunk 3 above)
Comment (quinn-dougherty) [reply setup.py]: setup.py is now from develop branch (just has selenium added to the list literal)

File: setup.py (hunk 2 above)
Comment (quinn-dougherty) [reply setup.py]: setup.py is now from develop branch (just has selenium added to the list literal)

**Review by jlubken — APPROVED** [review]
Go, go, go!

**Pull request merged**